docs/getting_started.md

```diff
@@ -9,7 +9,10 @@ following the instructions [here](https://docs.microsoft.com/en-us/azure/devops/
 If you already have an Azure DevOps organization, create a [new project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops).
 
 ## Clone or fork this repository
-Fork this repository within GitHub, or clone it into your Azure DevOps project.
+
+To initialize this repository with a custom project name refer [bootstraping doc.](../bootstrap/README.md\bootstrap\README.md)
```

Comment:
1. **Fork** this repository if there is a desire to contribute back to the repository  _else_
1. [Use this code template ](https://github.com/microsoft/MLOpsPython/generate) which copies the entire code base to your own GitHub location with the `git` commit history restarted.  This can be used for learning and following the guide.

If the desire is to use this project for your own machine learning code, follow the [bootstrap instructions](..\bootstrap\README.md) after the code template is complete.